src/Core/src/AwsClientFactory.php

```diff
@@ -138,6 +139,19 @@ public function dynamoDb(): DynamoDbClient
         return $this->serviceCache[__METHOD__];
     }
 
+    public function eventBridge(): EventBridgeClient
```

Comment:
Do not forget the changelog

Reply:
Sure. I've added a changelog entry.

Thanks